pkg/kubelet/events/event.go

```diff
@@ -77,6 +77,7 @@ const (
 	FailedPrepareDynamicResources        = "FailedPrepareDynamicResources"
 	PossibleMemoryBackedVolumesOnDisk    = "PossibleMemoryBackedVolumesOnDisk"
 	CgroupV1                             = "CgroupV1"
+	CgroupV2                             = "CgroupV2"
```

Comment:
Is this constant used anywhere?

Reply:
This is a `// kubelet event reason list`. So we put it here.

Reply:
Is there any event that uses this constant? All others are used in the `recorder.Event*` calls as far as I can see.

Reply:
Sorry, at first we want to add a warning event, and later, we changed to a log. So this can be removed.

Updated.